internal/workbench/desktop/models/src/appearance.rs

```diff
@@ -69,4 +103,101 @@ pub mod theming {
         is_default: bool,
         colors: Colors,
     }
+
+    fn transform_to_rgba<'de, D>(deserializer: D) -> Result<String, D::Error>
+    where D: Deserializer<'de>
+    {
+        let content = String::deserialize(deserializer)?;
+        let [r, g, b, a] = csscolorparser::parse(content.as_str())
+            .unwrap()
+            .to_rgba8();
+        let result = vec![r.to_string(),
+             g.to_string(),
+             b.to_string(),
+             format!("{:.2}", a as f64 / 255.0)];
+        Ok(result.join(","))
+
+    }
+
+    #[cfg(test)]
+    mod tests {
+        use super::*;
+        #[test]
+        fn test_rgba_theme() {
+            let theme = r###"
+{
+  "name": "Moss Pink",
+  "slug": "moss-pink",
+  "type": "pink",
+  "isDefault": false,
+  "colors": {
+    "primary": "rgba(0, 0, 0, 1)",
+    "sideBar.background": "rgba(234, 157, 242, 1)",
+    "toolBar.background": "rgba(222, 125, 232, 1)",
+    "page.background": "rgba(227, 54, 245, 1)",
+    "statusBar.background": "rgba(63, 11, 69, 1)",
+    "windowsCloseButton.background": "rgba(196, 43, 28, 1)",
+    "windowControlsLinux.background": "rgba(218, 218, 218, 1)",
+    "windowControlsLinux.text": "rgb(61, 61, 61, 1)",
+    "windowControlsLinux.hoverBackground": "rgb(209, 209, 209, 1)",
+    "windowControlsLinux.activeBackground": "rgb(191, 191, 191, 1)"
+  }
+}
+"###;
```

Comment:
What is being tested here? First, in future PRs, when we change token names, these tests will break. Additionally, we might have around 1,000 tokens in the future. Maintaining such tests would require significant effort. 

Moreover, in this test, we are essentially testing the function serde_json::from_str, which we shouldn’t be doing. Testing the functionality of a library should be the responsibility of its maintainers, not ours.

What we need to test in this case is whether we can correctly parse and transform various formats into the desired one. For this, we don’t need to test specific conversion functions. We should also include tests not only for successful cases but for incorrect ones as well.